### PR TITLE
Add CTA to Slack import doc

### PR DESCRIPTION
**Pull request by wiggin77**
#### Summary
Request from @jasonblais to add call-to-action link in Slack import docs.

#### Ticket Link
NONE

### CODE REVIEW

**Review by mickmister — APPROVED** [review]

File: source/onboard/migrate-from-slack.rst [hunk 1]
```diff
@@ -12,6 +12,10 @@ Migrating from Slack to Mattermost involves the following steps:
 4. `Convert the Slack import to Mattermost's format <#convert-slack-import-to-mattermost-s-bulk-export-format>`__
 5. `Import data into Mattermost <#import-data-into-mattermost>`__
 
+.. tip::
+
+  Looking for help migrating from Slack to Mattermost? [Contact us for Enterprise migration support](https://mattermost.com/contact-sales/).
```
Comment (wiggin77) [inline source/onboard/migrate-from-slack.rst:17]:
```suggestion
  Looking for help migrating from Slack to Mattermost? `Contact us for Enterprise migration support <https://mattermost.com/contact-sales/>`.
```

File: source/onboard/migrate-from-slack.rst [hunk 2]
```diff
@@ -12,6 +12,10 @@ Migrating from Slack to Mattermost involves the following steps:
 4. `Convert the Slack import to Mattermost's format <#convert-slack-import-to-mattermost-s-bulk-export-format>`__
 5. `Import data into Mattermost <#import-data-into-mattermost>`__
 
+.. tip::
+
+  Looking for help migrating from Slack to Mattermost? `Contact us for Enterprise migration support <https://mattermost.com/contact-sales/>`.
```
Comment (wiggin77) [inline source/onboard/migrate-from-slack.rst:17]:
```suggestion
  Looking for help migrating from Slack to Mattermost? `Contact us for Enterprise migration support <https://mattermost.com/contact-sales/>_`.
```

File: source/onboard/migrate-from-slack.rst [hunk 3]
```diff
@@ -12,6 +12,10 @@ Migrating from Slack to Mattermost involves the following steps:
 4. `Convert the Slack import to Mattermost's format <#convert-slack-import-to-mattermost-s-bulk-export-format>`__
 5. `Import data into Mattermost <#import-data-into-mattermost>`__
 
+.. tip::
+
+  Looking for help migrating from Slack to Mattermost? `Contact us for Enterprise migration support <https://mattermost.com/contact-sales/>_`.
```
Comment (wiggin77) [inline source/onboard/migrate-from-slack.rst:17]:
```suggestion
  Looking for help migrating from Slack to Mattermost? `Contact us for Enterprise migration support <https://mattermost.com/contact-sales/>`_.
```

**Comment by mickmister** [comment]
Here is the excerpt for reference:

http://mattermost-docs-preview-pulls.s3-website-us-east-1.amazonaws.com/6888/onboard/migrate-from-slack.html

CleanShot 2024-01-30 at 23 06 38

**Review by mickmister — APPROVED** [review]

**Review by amyblais — APPROVED** [review]

**Pull request merged**